### PR TITLE
W-15428649-Add-rescaffold-process-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by CTarzia — DISMISSED** [review]

File: modules/ROOT/pages/import-api-specification-exchange.adoc [hunk 1]
```diff
@@ -57,6 +57,15 @@ Manage the API specifications linked to Exchange in your project from the API sp
 [NOTE]
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Rescaffold an API Specification in a Mule Project
+
+After changing your API spec, such as adding a new endpoint, re-scaffold the spec to update your Mule project.
+
+. Right-click on the file.xml of your Mule project.
```
Comment (hannanelson) [inline modules/ROOT/pages/import-api-specification-exchange.adoc]: Is file.xml the actual name of the file? If so, use monospace.

Comment (hannanelson) [inline modules/ROOT/pages/import-api-specification-exchange.adoc]:
```suggestion
. Right-click `file.xml` in your Mule project.
```

File: modules/ROOT/pages/import-api-specification-exchange.adoc [hunk 2]
```diff
@@ -57,6 +57,15 @@ Manage the API specifications linked to Exchange in your project from the API sp
...
+. Select *API Specs*, the name of your Mule project (filename.xml), and *Generate Flows*.
```
Comment (hannanelson) [inline modules/ROOT/pages/import-api-specification-exchange.adoc]: filename.xml or file.xml?

Comment (hannanelson) [reply modules/ROOT/pages/import-api-specification-exchange.adoc]: Is Generate Flows a button? Should it be ", and then click Generate Flows"?

File: modules/ROOT/pages/import-api-specification-exchange.adoc [hunk 3]
```diff
@@ -57,6 +57,15 @@ Manage the API specifications linked to Exchange in your project from the API sp
 [NOTE]
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Rescaffold an API Specification in a Mule Project
+
+After changing your API spec, such as adding a new endpoint, re-scaffold the spec to update your Mule project.
```
Comment (hannanelson) [inline modules/ROOT/pages/import-api-specification-exchange.adoc]: "rescaffold"?

File: modules/ROOT/pages/import-api-specification-exchange.adoc [hunk 4]
```diff
@@ -57,6 +57,15 @@ Manage the API specifications linked to Exchange in your project from the API sp
...
+
+. Right-click on the file.xml of your Mule project.
+. Select *API Specs*, the name of your Mule project (filename.xml), and *Generate Flows*.
+
+If your project is successfully rescaffolded, a new flow is added to your implementation XML file.
```
Comment (hannanelson) [inline modules/ROOT/pages/import-api-specification-exchange.adoc]: Previously, you use "re-scaffold".

Comment (hannanelson) [reply modules/ROOT/pages/import-api-specification-exchange.adoc]: Any way to make these two verbs active?

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/import-api-specification-exchange.adoc (hunk 3 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/import-api-specification-exchange.adoc]: You're right! As always... :) Thanks!

File: modules/ROOT/pages/import-api-specification-exchange.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/import-api-specification-exchange.adoc]: The file name depends on the project name. If the project is called "MyFirstMuleProject" the file will be "myfirstmuleproject.xml". Let's see if my rewording works 🤞

Comment (Crispy-Salesforce) [reply modules/ROOT/pages/import-api-specification-exchange.adoc]: The sentence was reworded.

File: modules/ROOT/pages/import-api-specification-exchange.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/import-api-specification-exchange.adoc]: "Generate Flows" if one of the options in the context menu when you right click.

File: modules/ROOT/pages/import-api-specification-exchange.adoc (hunk 4 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/import-api-specification-exchange.adoc]: Let's see if this rewording works.

File: modules/ROOT/pages/import-api-specification-exchange.adoc [hunk 5]
```diff
@@ -57,6 +57,15 @@ Manage the API specifications linked to Exchange in your project from the API sp
...
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Rescaffold an API Specification in a Mule Project
+
+After changing your API spec, such as adding a new endpoint, rescaffold the spec to update your Mule project.
+
+. Right-click on the XML file of your Mule project.
+. Select *API Specs*, the name of your Mule project, and *Generate Flows*.
+
+If the rescaffolding process of your project is successful, your implementation XML file adds a new flow.
```
Comment (hannanelson) [inline modules/ROOT/pages/import-api-specification-exchange.adoc]:
```suggestion
If the rescaffolding process of your project is successful, your implementation XML file includes a new flow. 
```

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/import-api-specification-exchange.adoc (hunk 5 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/import-api-specification-exchange.adoc]: OK! Thanks!

**Review by hannanelson — APPROVED** [review]

**Pull request merged**